README.rst

```diff
@@ -0,0 +1,65 @@
+======================================
```

Comment:
Nit: trim the number of equal signs to fit the length of the thing being header-ed.

Reply:
Done.